go-controller/pkg/node/gateway_shared_intf_linux.go

```diff
@@ -93,6 +93,21 @@ func setupLocalNodeAccessBridge(nodeName string, subnets []*net.IPNet) error {
 	return nil
 }
 
+// deletes the local bridge used for DGP and removes the corresponding iface
+func deleteLocalNodeAccessBridge() error {
+	if err := util.LinkDelete(localnetGatewayNextHopPort); err != nil {
```

Comment:
I0310 15:55:36.744132   63838 node.go:253] System upgrade detected, falling back to to legacy K8S Service via mp0
I0310 15:55:36.744417   63838 node.go:399] Waiting for OVN Masters to upgrade to switchover K8S Service route
I0310 15:58:01.809894   63838 node.go:410] Detected Master upgrade, updating K8S Service route
I0310 15:58:01.810649   63838 helper_linux.go:73] Found default gateway interface br-ex 10.0.192.1
I0310 15:58:01.810828   63838 node.go:426] Successfully updated Kubernetes service route towards OVS
E0310 15:58:01.811001   63838 node.go:429] Error while removing Local Node Access Bridge, error: failed to remove link ovn-k8s-gw0, error: operation not supported

Probably would need to set the port down first, but this isn't necessary. I found just deleting the ovs bridge removes this interface so will remove this change.